packages/slice-machine/lib/env/resolveAliases.ts

```diff
@@ -12,7 +12,10 @@ type PackageWithModuleAliases = JsonPackage & {
 const isAPackageHasModuleAliases = (
   jsonPackage: JsonPackage | PackageWithModuleAliases
 ): jsonPackage is PackageWithModuleAliases => {
-  return jsonPackage.hasOwnProperty("_moduleAliases");
+  return (
+    typeof jsonPackage === typeof {} &&
```

Comment:
I don't understand why this file keeps giving eslint errors on different branches and the solution is always different 😂 .

```suggestion
    typeof jsonPackage === 'object' &&
```

Reply:
Probably a minor/path update to a package